exasol_advanced_analytics_framework/resources/outputs/create_query_loop.sql

```diff
@@ -12,380 +12,224 @@ CREATE OR REPLACE LUA SCRIPT "AAF_RUN_QUERY_HANDLER"(json_str) RETURNS TABLE AS
 
```

Comment:
This file is really huge. Can't it be divided in several sql files?

Reply:
Generated file

Reply:
In this case it would be good to have a header in the file explaining that it is generated.